engine/src/main/resources/templates/ResourceOptimizationService/dailyEmailBody.html

```diff
@@ -0,0 +1,26 @@
+{#include ResourceOptimizationService/insightsEmailBody}
+{#content-title}Daily Resource Optimization Service Summary - {action.context.start_time.toStringFormat()}{/content-title}
+{#content-body}
+<tr>
+    <td class="rh-content__block">
+        <p>Resource Optimization Service has identified <b> {action.context.system_count}</b> system{#if action.context.system_count > 1}s{/if} with new suggestion{#if action.context.system_count > 1}s{/if}.
+        </p>
+    </td>
+</tr>
+<tr>
+    <td class="rh-content__block">
+        <table align="center">
+            <tr>
+                <td class="rh-cta-link" align="center">
+                    <a target="_blank" href="{environment.url}/insights/ros">
+                                <span>
+                                  Go to Resource Optimization Service
```

Comment:
Customers currently see `Resource Optimization` without `Service` everywhere in console.redhat.com. Would it make sense to drop `Service` from all `Resource Optimization Service` occurrences in this template? You may want to ask your PM about that.
```suggestion
                                  Go to Resource Optimization
```

Reply:
We need to confirm this from our UX. He is on PTO. We can wait for now.